pkg/kernel/kernel.go

```diff
@@ -1,28 +1,28 @@
-package main
+package kernel
 
 import (
 	"regexp"
 	"strconv"
 )
 
-type kernelVersion struct {
+type KernelVersion struct {
```

Comment:
this stutters `kernel.KernelVersion`. Maybe name it just `Version`, so it becomes `kernel.Version`?